templates/submission/status-testcases.html

```diff
@@ -144,7 +144,7 @@ <h3>{{ _('Execution Results') }}</h3>
                 {% else %}
                     <b>{{ _('Final score:') }}</b>
                 {% endif %}
-                {{ submission.case_points|floatformat(0) }}/{{ submission.case_total|floatformat(0) }}
+                {{ submission.case_points|floatformat(3) }}/{{ submission.case_total|floatformat(0) }}
```

Comment:
You should change the case points floatformat value to -3. This will remove trailing zeros for values that are supposed to be integers.